src/sql/src/statements.rs

```diff
@@ -238,11 +238,14 @@ fn parse_column_default_constraint(
 // a column is not nullable but has a default value null.
 /// Create a `ColumnSchema` from `ColumnDef`.
 pub fn column_def_to_schema(column_def: &ColumnDef, is_time_index: bool) -> Result<ColumnSchema> {
-    let is_nullable = column_def
-        .options
-        .iter()
-        .any(|o| matches!(o.option, ColumnOption::Null));
-
+    let is_nullable = if column_def.options.is_empty() {
```

Comment:
It's not enough. In fact, we want the `nullable` as the default property, it's the same that we can't find any `ColumnOption::NotNull` option in `column_def.options`.

Reply:
(Null)